libs/indibase/indiccd.h

```diff
@@ -728,8 +728,8 @@ class CCD : public DefaultDevice, GuiderInterface
         INDI::PropertyNumber ScopeInfoNP {2};
         enum
         {
-            FocalLength,
-            Aperture
+            FOCAL_LENGTH,
+            APERTURE
```

Comment:
is there a reason for this change? any other drivers using these enums?

Reply:
It is used also in `guide_simulator.cpp` and `ccd_simulator.cpp`. Checking other parts of the code, it seems to be conform to use capital letter enums definitions. So far all
```
enum {
 CAPITAL_LETTERS
 ...
 ...
}

```
have this capital letter pattern.